rules/cloud/okta/okta_impossible_travel_detected.yml

```diff
@@ -0,0 +1,28 @@
+title: Okta Impossible Travel Detected
+id: 7e302f6b-533d-4f57-8041-0480773744ff
+description: Detects when impossible travel is detected by Okta
+author: Aidan Bracher @brachera
+status: experimental
+date: 2022/09/26
+references:
+    - https://help.okta.com/en-us/Content/Topics/Security/behavior-detection/about-behavior-detection.htm
+    - https://help.okta.com/en-us/Content/Topics/Security/behavior-detection/logs-behavior-detection.htm
+logsource:
+  product: okta
+  service: okta
+detection:
+    selection:
+        okta.event_type: 
```

Comment:
HI,
In other rule get `eventtype` and `displaymessage` for the field name.

Reply:
I don't understand that comment

Reply:
I don't have okta but in the other rule find `eventtype` and `displaymessage` .
missing the question: 
Are you sure of the name?

Reply:
Ping @brachera